conf/services/agents.yml

```diff
@@ -0,0 +1,15 @@
+serviceName: Agents Online Self-Serve
+serviceHeaderName: Agent services
+serviceDescription: |
+  Agents Online Self-Serve provides a client tax dashboard where agents can view the PAYE liabilities and payments for
+  their clients, amend client details online, access client services such as online filing and payment plans, confirm
+  that they still act for a given client, and access old portal PAYE for agents features such as Tax codes.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /agents
```

Comment:
The service url is /agent if you look in prod routes in agent-frontend

Reply:
Also maybe rename the yml to be agent.yml

Reply:
done